decanter_ai_sdk/experiment.py

```diff
@@ -28,6 +28,7 @@ class Experiment(BaseModel):
     gp_table_id: str
     holdout: Dict[str, str]
     holdout_percentage: float
+    hyperparameters: Dict[str,Any]
```

Comment:
It should be `Dict[Any]`, this object has different typing. Like: str, bool, dict, ...